Data/Sys/ApprovedInis.json

```diff
@@ -932,10 +932,6 @@
     "title": "Region Select",
     "AD12237401ABE9FE4A545AADB5C5AE10355E2076": "RSAPatch"
   },
-  "RELJAB": {
-    "title": "SegaBoot",
-    "130F3594CAB57B85616F95C7126F4748AAC5867D": "DI Seed Blanker"
-  },
```

Comment:
If you update this file, you also need to update `APPROVED_LIST_HASH`, otherwise RetroAchievements will break:

https://github.com/dolphin-emu/dolphin/blob/f8bf35e6f01e0165e2877031bf19cd17e71e61e2/Source/Core/Core/AchievementManager.h#L82-L84

Ideally we should have a unit test to catch this, but we currently don't.

Reply:
Fixed.